gradle/build-logic/convention/src/main/kotlin/KotlinMultiplatformConventionPlugin.kt

```diff
@@ -23,6 +23,9 @@ class KotlinMultiplatformConventionPlugin : Plugin<Project> {
     extensions.configure<KotlinMultiplatformExtension> {
       applyDefaultHierarchyTemplate()
       jvm("desktop")
+      iosArm64()
+      iosX64()
+      iosSimulatorArm64()
```

Comment:
Not familiar with publishing KMP libraries for iOS. I think we may need the equivalent to `publishLibraryVariants` but for iOS?

Reply:
Yea I'm not sure either. Let's try merging this and seeing if a SNAPSHOT version gets published correctly for iOS.

Reply:
Any ideas on the failures on this PR? The error mentions a missing android target, but I did not modify that 🤔

Reply:
Sorry I'm just seeing this. Let me try building the project on my machine.